### PR TITLE
DOC Update README.md for CMS 5

### PR DESCRIPTION
**Pull request by ChloeSartorelli**
## Issue

- https://github.com/silverstripe/.github/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,9 @@ The Asset Admin module provides a interface for managing files within Silverstri
 
 ## Installation
 
+```sh
+composer require silverstripe/asset-admin
 ```
-$ composer require silverstripe/asset-admin
-```
-
-You'll also need to run `dev/build`.
```
Comment (sabina-talipova) [inline README.md:16]: I've already merge couple another PR with similar changes, but small question.
Is it correct that we remove recommendation about `dev/build`?

Comment (ChloeSartorelli) [reply README.md:16]: Yeah I noticed this was present in a couple of modules but not in others. I don't think it really matters too much one way or another... it's pretty common knowledge and is documented in the main ss docs (or at least should be) that this is needed any time database schema is updated. And I'd argue nobody should be installing modules without knowing what sort of affect that module will have.

tl;dr: 🤷‍♀️

**Review by sabina-talipova — APPROVED** [review]
LGFM
